src/components/Animal/Animal.js

```diff
@@ -1,3 +1,13 @@
-export default function Animal() {
-  return <></>;
+import './Animal.css';
+
+export default function Animal({ animal }) {
+  return (
+    <>
+      <div className="animal" style={{ top: animal.top, left: animal.left }}>
+        <p className="name">{animal.name}</p>
+        {animal.says}
+        <img src={`${process.env.PUBLIC_URL}/animals/${animal.type}.svg`} />
```

Comment:
You don't need the process.env.PUBLIC_URL.

`animals/${props.type}.svg` should work